receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_scraper.go

```diff
@@ -29,40 +29,36 @@ import (
 
 // scraper for FileSystem Metrics
 type scraper struct {
-	config    *Config
-	includeFS filterset.FilterSet
-	excludeFS filterset.FilterSet
+	config   *Config
+	fsFilter fsFilter
 
 	// for mocking gopsutil disk.Partitions & disk.Usage
 	partitions func(bool) ([]disk.PartitionStat, error)
 	usage      func(string) (*disk.UsageStat, error)
 }
 
+type fsFilter struct {
```

Comment:
Nit: I think this type should be defined in `config.go` since that's where the constructor is.